vespa/application.py

```diff
@@ -397,33 +397,37 @@ def query_batch(
                 **kwargs,
             )
 
-    def feed_data_point(self, schema: str, data_id: str, fields: Dict) -> VespaResponse:
+    def feed_data_point(self, schema: str, data_id: str, fields: Dict, namespace: str=None) -> VespaResponse:
         """
         Feed a data point to a Vespa app.
 
         :param schema: The schema that we are sending data to.
         :param data_id: Unique id associated with this data point.
         :param fields: Dict containing all the fields required by the `schema`.
+        :param namespace: The namespace that we are sending data to.
         :return: Response of the HTTP POST request.
         """
+        if not namespace:
+            namespace = schema
+
         with VespaSync(app=self) as sync_app:
             return sync_app.feed_data_point(
-                schema=schema, data_id=data_id, fields=fields
+                namespace=namespace, schema=schema, data_id=data_id, fields=fields
             )
 
-    def _feed_batch_sync(self, schema: str, batch: List[Dict]):
+    def _feed_batch_sync(self, namespace: str, schema: str, batch: List[Dict], **kwargs) -> VespaResponse:
```

Comment:
1) Let's follow a pattern of adding the `namespace` argument at the end of the parameter list.
2) Why add `**kwargs` here?